script/collections/brand-consolidation.json

```diff
@@ -265,7 +265,7 @@
     }
   },
   "educationYellowRibbonProgram": {
-    "pattern": "education/about-gi-bill-benefits/yellow-ribbon-program/*.md",
+    "pattern": "education/about-gi-bill-benefits/post-9-11/yellow-ribbon-program/*.md",
```

Comment:
Does this address https://github.com/department-of-veterans-affairs/vets.gov-team/issues/13960?

Reply:
@ncksllvn I thought that would fix it but it doesn't. There is something else going on either in the build system or the templating system.